test/timecop_test.rb

```diff
@@ -689,8 +689,106 @@ def test_thread_safe_timecop_returns_after_block
     Timecop.thread_safe = false
   end
 
+  if RUBY_VERSION >= '2.1.0'
+    def test_process_clock_gettime_monotonic
+      Timecop.freeze do
+        assert_same monotonic, monotonic, "CLOCK_MONOTONIC is not frozen"
+      end
+
+      current = monotonic
+      Timecop.freeze(-0.5) do
+        assert monotonic < current, "CLOCK_MONOTONIC is not traveling back in time"
+      end
+    end
+
+    def test_process_clock_gettime_monotonic_nested
+      Timecop.freeze do
+        parent = monotonic
+
+        sleep(0.1)
+
+        Timecop.freeze(0.5) do
+          child = monotonic
+          assert_times_effectively_equal(child, parent + 0.5, 0.001, "Nested freeze not working for monotonic time")
+        end
+      end
+    end
+
+    def test_process_clock_gettime_monotonic_travel
+      current = monotonic
+      Timecop.travel do
+        refute_same monotonic, monotonic, "CLOCK_MONOTONIC is frozen"
+        sleep 0.5
+        assert monotonic > current, "CLOCK_MONOTONIC is not moving forward"
+      end
```

Comment:
I'm not sure a sleep is necessary, but if it is, I would put it _before_ the travel, to ensure that it does tick forward.

And similar to above, I think 10ms is plenty to make it do at least one tick.
```suggestion
      current = monotonic
      sleep 0.01
      Timecop.travel do
        refute_same monotonic, monotonic, "CLOCK_MONOTONIC is frozen"
        assert monotonic > current, "CLOCK_MONOTONIC is not moving forward"
      end
```

You could define a `TIME_EPSILON = 0.01` constant in the test suite to capture the intent